pynetdicom/dimse_messages.py

```diff
@@ -302,6 +304,16 @@ def decode_msg(self, primitive):
                         _MESSAGE_TYPES[self.command_set.CommandField][1]
                     )
 
+                    if (
+                        self.__class__ == C_STORE
+                        and _config.STORE_RECV_CHUNKED_DATASET_PATH
+                    ):
+                        self._data_set_file = (
+                            _config.STORE_RECV_CHUNKED_DATASET_PATH
+                            / get_ident()
+                            / f"{self.command_set.AffectedSOPInstanceUID}.dcm"
+                        )
```

Comment:
TODO: Write file_meta

Reply:
Hm, its pretty tricky to write the file_meta here since we can't link the `context_id` to a real context from the association.